### PR TITLE
Added AXE

### PR DESCRIPTION
**Pull request by charlesrocket**
https://github.com/AXErunners/axe/

### CODE REVIEW

**Comment by charlesrocket** [comment]
fixed the conflict

File: lib/coins/axe.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+var main = Object.assign({}, {
+  hashGenesisBlock: '0x00000848c4c426413b0a42d4d127085f903670f51aca7b177d2c004867bc50af',
+  port: 29057,
+  portRpc: 29058,
+  protocol: {
+  },
+  versions: {
+    bip32: {
+    },
```
Comment (fanatid) [inline lib/coins/axe.js]: constants should be set

File: lib/coins/axe.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    bip44: 7,
```
Comment (fanatid) [inline lib/coins/axe.js]: 7 belongs to namecoin
I do not see AXE here: https://github.com/satoshilabs/slips/blob/master/slip-0044.md

File: lib/coins/axe.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    private: 0xCB,
+    public: 0x4B,
+    scripthash: 0x55
+  }
+}, common)
+
+module.exports = {
+  main: main,
+  test: null
```
Comment (fanatid) [inline lib/coins/axe.js]: `test: null` line is useless

File: package.json [hunk 4]
```diff
@@ -19,7 +19,8 @@
     "monacoin",
     "zcash",
     "vertcoin",
-    "qtum"
+    "qtum",
+    "axe"
```
Comment (fanatid) [inline package.json]: alphabetical order is preferred

**Comment by fanatid** [comment]
Please squash your commits.

**Comment by fanatid** [comment]
Also, please add url to source file (with commit hash if available) where parameters can be reviewed.

**Pull request closed without merging**